osu.Game/Online/RealtimeMultiplayer/MultiplayerUserState.cs

```diff
@@ -5,6 +5,16 @@ namespace osu.Game.Online.RealtimeMultiplayer
 {
     public enum MultiplayerUserState
     {
+        /// <summary>
+        /// The user has not downloaded the beatmap yet.
+        /// </summary>
+        BeatmapNotDownloaded,
```

Comment:
have you looked how this is serialized, and whether inserting at the *start* of the enum will break everything?

Reply:
oh..., my bad.